src/types/response.rs

```diff
@@ -156,4 +156,14 @@ impl PyResponse {
             .into_py(py);
         Ok(())
     }
+
+    pub fn set_cookie(&mut self, py: Python, key: &str, value: &str) -> PyResult<()> {
```

Comment:
```
impl PyResponse {
    pub fn set_cookie(&mut self, name: &str, value: &str, path: Option<&str>, expires: Option<&str>, http_only: bool, secure: bool) {
        let mut cookie_value = format!("{}={}", name, value);
        if let Some(path) = path {
            cookie_value.push_str(&format!("; Path={}", path));
        }
        if let Some(expires) = expires {
            cookie_value.push_str(&format!("; Expires={}", expires));
        }
        if http_only {
            cookie_value.push_str("; HttpOnly");
        }
        if secure {
            cookie_value.push_str("; Secure");
        }

        self.headers.insert("Set-Cookie".to_string(), cookie_value);
    }
}

```

Reply:
Why not something like the above?

Reply:
@sansyrox sure, if you want it to be done so, I will update. we should probably give a `#get_cookie` function too, IMO.

(I implemented it the way I did cuz of the example code provided in the issue).

Reply:
Hey @VishnuSanal , added a suggestion in near the exact line of concern.